st3/sublime_lib/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .output_panel import OutputPanel  # noqa: F401
+from .output_panel import Panel, OutputPanel  # noqa: F401
```

Comment:
Should probably rename this module to just `panel`, i.e. the more general name.

Reply:
Done.